include/mbedtls/config.h

```diff
@@ -615,6 +615,22 @@
  */
 //#define MBEDTLS_AES_ONLY_128_BIT_KEY_LENGTH
 
+/**
+ * \def MBEDTLS_AES_ONLY_ENCRYPT
```

Comment:
Minor - the name is fine, but it makes more sense in English (to me at least) as `MBEDTLS_AES_ENCRYPT_ONLY`. I suggest we change this in upstreaming.